clarifai/client/dataset.py

```diff
@@ -282,46 +284,44 @@ def _data_upload(self, dataset_obj: ClarifaiDatasetType) -> None:
           progress.update()
 
   def upload_dataset(self,
-                     task: str,
-                     split: str,
                      module_dir: str = None,
-                     dataset_loader: str = None,
-                     chunk_size: int = 128) -> None:
+                     dataloader: Type[ClarifaiDataLoader] = None,
+                     batch_size: int = 32) -> None:
     """Uploads a dataset to the app.
 
     Args:
-      task (str): task type(text_clf, visual-classification, visual_detection, visual_segmentation, visual-captioning)
-      split (str): split type(train, test, val)
       module_dir (str): path to the module directory
-      dataset_loader (str): name of the dataset loader
-      chunk_size (int): chunk size for concurrent upload of inputs and annotations
+      dataloader (Type[ClarifaiDataLoader]): ClarifaiDataLoader object
```

Comment:
Nit: Two separate words, so maybe should keep snake case.
```suggestion
      data_loader (Type[ClarifaiDataLoader]): ClarifaiDataLoader object
```

Reply:
I think `dataloader` is the usual convention, pytorch etc, follows the same